# EMERGE Biology Integration Institute

Predictive understanding of ecosystem response to change has become a pressing societal need in the Anthropocene, and requires integration across disciplines, spatial scales, and timeframes. Developing a framework for understanding how different biological systems interact over time is a major challenge in biology. The National Science Foundation-funded EMergent Ecosystem Responses to ChanGE (EMERGE) Biology Integration Institute aims to develop such a framework by integrating research, training, and high-resolution field and laboratory measurements across 15 scientific subdisciplines–including ecology, physiology, genetics, biogeochemistry, remote sensing, and modeling–across 14 institutions, in order to understand ecosystem-climate feedbacks in Stordalen Mire, a thawing permafrost peatland in arctic Sweden. Rapid warming in the Arctic is driving permafrost thaw, and new availability of formerly-frozen soil carbon for cycling and release to the atmosphere, representing a potentially large but poorly constrained accelerant of climate change. This material is based upon work supported by the National Science Foundation under Grant Number 2022070.

Listed below are a number of the tools that members have developed for better understanding and integration of these datasets.


# EMERGE tools
|                               Tool                                |                                         Description                                         |                                 Developers                                 |                 Citation                 |
|-------------------------------------------------------------------|---------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|------------------------------------------|
|[CoverM](https://github.com/wwood/CoverM)                          |Metagenomic coverage calculator / BAM file generator                                         |Ben Woodcroft (CMR)                                                         |                                          |
|[Lorikeet](https://github.com/rhysnewell/Lorikeet)                 |Microbial strain resolver, coverage calculator, variant caller, selective pressure calculator|Rhys Newell (CMR)                                                           |                                          |
|[Rosella](https://github.com/rhysnewell/rosella)                   |Metagenomic binning and bin refinement tool                                                  |Rhys Newell (CMR)                                                           |                                          |
|[Aviary (incorporated SlamM)](https://github.com/rhysnewell/aviary)|Microbial genome recovery pipeline with novel methods for long/short read assembly           |Rhys Newell (CMR)                                                           |                                          |
|[Galah](https://github.com/wwood/galah)                            |Genome dereplication                                                                         |Ben Woodcroft (CMR)                                                         |                                          |
|[Kingfisher](https://github.com/wwood/kingfisher-download)         |Public sequence and metadata gatherer                                                        |Ben Woodcroft (CMR)                                                         |                                          |
|[SingleM](https://github.com/wwood/singlem)                        |De-novo OTUs from shotgun metagenomes                                                        |Ben Woodcroft (CMR)                                                         |                                          |
|[GraftM](https://github.com/geronimp/graftM)                       |Meta-omic tool that identifies and classifies marker and functional genes                    |Ben Woodcroft (CMR)                                                         |https://doi.org/10.1093/nar/gky174        |
|[[DRAM](https://github.com/shafferm/DRAM)](https://github.com/WrightonLabCSU/DRAM)                           |Annotates MAGs and summarizes metabolic potential                                            |Mikayla Borton (CSU), Mike Shaffer (CSU), Kelly Wrighton                    |https://doi.org/10.1093/nar/gkaa621       |
|[Phylogenetic Null Modeling](https://github.com/sljarvis2)         |partitioning variation in phylogenetic data and attributing to assembly processes            |Stacey Doherty (UNH alum), Jessica Ernakovich (based on Stegen et al., 2013)|                                          |
|[vConTACT2 ](https://bitbucket.org/MAVERICLab/vcontact2/)          |Classifies and clusters viral sequences into approx. genus groups                            |Ben Bolduc (OSU), Sullivan Lab                                              |https://doi.org/10.1038/s41587-019-0100-8 |
|[vConTACT3 ](https://bitbucket.org/MAVERICLab/vcontact3/)          |Classifies and clusters viral sequences into approx. genus groups (updated vConTACT3)        |Ben Bolduc (OSU), Sullivan Lab                                              |https://doi.org/10.1038/s41587-019-0100-8 |
|[VirSorter1](https://github.com/simroux/VirSorter)                 |Identifies viral sequences in microbial and viral sequence data                              |Simon Roux (JGI), Sullivan Lab                                              |https://doi.org/10.7717/peerj.985         |
|[VirSorter2](https://github.com/jiarong/VirSorter2)                |As VirSorter1, but uses ML and expands viral types detected                                  |Jiarong Guo (OSU), Sullivan Lab                                             |https://doi.org/10.1186/s40168-020-00990-y|
|[MetaPop](https://github.com/metaGmetapop/metapop)                 |calculates macro- and micro-diversity metrics                                                |Ann Gregory (OSU alum), Sullivan Lab                                        |https://doi.org/10.1186/s40168-022-01231-0|
|[ecosys](https://github.com/jinyun1tang/ECOSYS/)                   |site, landscape, and continental-scale land model                                            |Riley, Zhen, Grant                                                          |                                          |
|[CheckM 2](https://github.com/chklovski/CheckM2)                   |Microbial genome Q/A                                                                         |Alex Chklovski (CMR)                                                        |https://doi.org/10.1038/s41592-023-01940-w|
